src/server/pfs/fuse/server_test.go

```diff
@@ -1026,8 +1026,8 @@ func TestProjects(t *testing.T) {
 
 		require.NoError(t, json.NewDecoder(resp.Body).Decode(&projectData))
 		require.Equal(t, len(projectData), 3)
-		require.Equal(t, projectData[0].Project.Name, emptyProjectName)
+		require.Equal(t, projectData[0].Project.Name, pfs.DefaultProjectName)
```

Comment:
I just realized the order is not deterministic here because your `SELECT name, description, created_at FROM core.projects` doesn't order by anything. I think you should expose the ordering to the user, or by default order by `id`.